wiki/Gameplay/Judgement/osu!taiko/fr.md

```diff
@@ -1,34 +1,57 @@
----
-outdated_translation: true
-outdated_since: 1d2e724ee8a7cb4f313653a07c28af43c8443ea4
----
-
 # Système de jugement d'osu!taiko
 
-## Valeurs de hit
+## Jugements
+
+Un **jugement**, ou **résultat de hit** (hit result), est le résultat de l'interaction avec un [objet](/wiki/Gameplay/Hit_object) pendant sa fenêtre de hit. Le score et la précision sont calculés en fonction des jugements reçus.
+
+| Image | Nom | [Valeur de hit](/wiki/Gameplay/Score/ScoreV1/osu!taiko) | [Précision](/wiki/Gameplay/Accuracy#osu!taiko) | Erreur de hit maximum (ms) |
+| :-: | :-: | --: | --: | :-- |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit300g.png) ![](/wiki/shared/judgement/osu!taiko/taiko-hit300.png) | GREAT | 300 | 100% | `50 - 3 × OD` |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit100k.png) ![](/wiki/shared/judgement/osu!taiko/taiko-hit100.png) | OK | 150 | 50% | `120 - 8 × OD` si OD ≤ 5, et `110 - 6 × OD` si OD ≥ 5 |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit0.png) | MISS | 0 | 0% | `135 - 8 × OD` si OD ≤ 5, et `120 - 5 × OD` si OD ≥ 5 |
+
+La fenêtre de hit dépend de la [difficulté globale (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximum`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre. La fenêtre MISS compare par exception `erreur de hit ≤ erreur de hit maximum` à la place.
+
+L'erreur de hit est arrondie et les valeurs maximales d'erreur de hit sont arrondie à l'entier le plus proche, ce qui signifie que pour les GREAT et les OK, les fenêtres peuvent être jusqu'à 1,5 ms plus courtes des deux côtés, tandis que les fenêtres de miss peuvent être jusqu'à 0,5 ms plus courtes ou plus large des deux côtés, par rapport à ce que les formules suggèrent.
+
+## Mécaniques de jugement
+
+### Notes petites/larges
+
+Les petites et les notes larges sont jugées par un GREAT, un OK ou un MISS en fonction de la précision avec laquelle elles sont hit. Hit une note avant la fenêtre MISS n'a aucun effet, et ne pas frapper une note entraînera un MISS une fois la fenêtre MEH passée. Le fait de hit la mauvaise touche pour la couleur de la note entraînera également un MISS.
```

Comment:
```suggestion
Les petites et les notes larges sont jugées par un GREAT, un OK ou un MISS en fonction de la précision avec laquelle elles sont hit. Hit une note avant la fenêtre MISS n'a aucun effet, et ne pas frapper une note entraînera un MISS une fois la fenêtre MEH passée. Le fait de hit la mauvaise touche pour la couleur de la note entraînera également un MISS.
```

@Walavouchey does the MEH judgement exist on osu!taiko? I'm not sure, as there's no mention of it in the tables.